src/task/JoinAndReceiveIndexTask.ts

```diff
@@ -71,6 +71,10 @@ export default class JoinAndReceiveIndexTask extends BaseTask {
           if (event.message.type === SdkSignalFrame.Type.JOIN_ACK) {
             // @ts-ignore: force cast to SdkJoinAckFrame
             const joinAckFrame: SdkJoinAckFrame = event.message.joinack;
+            if (joinAckFrame && joinAckFrame.videoSubscriptionLimit) {
+              videoSubscriptionLimit = joinAckFrame.videoSubscriptionLimit;
+            }
+            context.videoSubscriptionLimit = videoSubscriptionLimit;
```

Comment:
nit: We could shorten this to joinAckFrame?.videoSubscriptionLimit ? joinAckFrame.videoSubscriptionLimit : 25;

Reply:
I tried doing this and the single line went too long. so would like to continue to keep it as is.